IVAN-MARAKHOVSKII/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="../node_modules/normalize.css/normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>New Jersey</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="grid">
+      <div class="wheatherWidget">
+        <div class="refreshIcon">
+          <img src="refresh.png" alt="" />
+        </div>
+        <div class="wheatherImage">
+          <img src="cloud_icon.png" />
+        </div>
+        <div class="infoBlock">
+          <img src="tempIcon.png" class="tempIcon" />
+          <div class="infoText">
+            <span class="infoTextTop">Cloudy Skies</span><br /><span
+              class="infoTextBottom"
+              >Sicklerville, New Jersey</span
+            >
+          </div>
+          <div class="dayInfo">
+            <span class="topDayText">MAY</span>
+            <span class="bottomDayText">21</span>
+          </div>
+        </div>
+        <div class="stats">
+          <div class="firstPage">
+            <div class="wind">
+              <img src="wind.png" alt="" />
+              <span><b>2 MPH</b></span>
+            </div>
+            <div class="humidity">
+              <img src="waterdrop.png" alt="" />
+              <span><b>33</b>%</span>
```

Comment:
It's convenient to avoid tags like `<b>` and change appearance of the text through css